src/shared/common/components/document-pane.jsx

```diff
@@ -12,13 +12,42 @@ import status from '../../helpers/api-status.js';
 import { Link } from 'react-router-dom';
 import sortByTimeStamp from '../../helpers/sortByTimeStamp.js';
 
+class DocumentPanelHelpers {
```

Comment:
No real need for this to be a class. 

I'd be happy with these helper being moves to src/shared/helpers under a filename `DocumentPanelHelpers.js`. The functions can then be exported and imported into the component. Therefore we have 
[https://en.wikipedia.org/wiki/Separation_of_concerns](separation of concerns). Tests then could also be moved to src/shared/helpers/__tests__.

Reply:
The functions would no longer need to be static either.

Reply:
Thanks @LiamMacP, I've moved them out into a separate file and done the same with the unit tests.